.github/dependabot.yml

```diff
@@ -0,0 +1,55 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+      day: "friday"
+    labels:
+      - "part:tooling"
+      - "type:tech-debt"
+    versioning-strategy: auto
+    open-pull-requests-limit: 10
+    groups:
+      required:
+        dependency-type: "production"
+        update-types:
+          - "minor"
+          - "patch"
+      optional:
+        dependency-type: "development"
+        update-types:
+          - "minor"
+          - "patch"
+    ignore:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+      day: "friday"
+    labels:
+      - "part:tooling"
+      - "type:tech-debt"
```

Comment:
Minor, but since we added it recently to repo-config, maybe is good to add it here from the start too:

```yaml
    groups:
      compatible:
        update-types:
          - "minor"
          - "patch"
      artifacts:
        patterns:
          - "actions/*-artifact"
```